frappe/public/js/frappe/ui/page.js

```diff
@@ -414,7 +414,7 @@ frappe.ui.Page = class Page {
 			parent.parent().removeClass("hide");
 		}
 
-		let $link = this.is_in_group_button_dropdown(parent, 'li > a.grey-link', label);
+		let $link = this.is_in_group_button_dropdown(parent, 'li > a.grey-link > span', label);
```

Comment:
This is breaking stuff. `page.add_dropdown_item` is expected to return a jQuery object, not boolean value.

Reply:
`add_menu_item` returns boolean because of this and causes this error:
![image](https://user-images.githubusercontent.com/16315650/173176344-e64ad2e2-c1ee-4b64-9c71-ed966965e42b.png)